Python/fileutils.c

```diff
@@ -2196,7 +2199,7 @@ _Py_find_basename(const wchar_t *filename)
 wchar_t *
 _Py_normpath(wchar_t *path, Py_ssize_t size)
 {
-    if (!path[0] || size == 0) {
+    if (path == NULL || !path[0] || size == 0) {
```

Comment:
alternatively this could be handled at callsite.

Reply:
IMO Adding assert to check path is not NULL will be enough.

Reply:
Are we assuming it is impossible for `_Py_join_relfile` to fail? Or do you mean callers should ensure it is not NULL?

Reply:
> callers should ensure it is not NULL

Since the api is private API, caller should care this before using it.

Reply:
I made sure callers perform the NULL check and added an assert.